CHANGELOG.md

```diff
@@ -81,6 +81,11 @@
 * [BUGFIX] Prevent panic at start if the http_prefix setting doesn't have a valid value. #3796
 * [BUGFIX] Memberlist: fixed panic caused by race condition in `armon/go-metrics` used by memberlist client. #3724
 
+## Blocksconvert
+
+* [ENHANCEMENT] Scanner: Implement for DynamoDB (v9 schema only). #3828
```

Comment:
Cortex release 1.8.0 is now in progress. Could you please rebase master and move the CHANGELOG entry under the master / unreleased section?

Reply:
@bboreham Could you address this, so we can merge it, please?